### PR TITLE
Config - Make a single configuration for version using an INI file

### PR DESCRIPTION
**Pull request by Gustry**
Linked to #2972

Need to add a new step when releasing a version to edit this new file with latest version of the Lizmap server plugin version

### CODE REVIEW

**Comment by laurentj** [comment]
Anyway, you should not use `\Jelix\IniFile\IniModifier` to read an ini file, it takes many resources. Just use `$ini = jIniFile::read($filename); $val = $ini['versions']['qgisMinimumVersionRequired']`

**Comment by Gustry** [comment]
Thanks, updated.
I was following @mdouchin comment in https://github.com/3liz/lizmap-web-client/pull/3132#issuecomment-1199376866 about the new file. But indeed, it looks ok like this.

File: lizmap/app/system/mainconfig.ini.php [hunk 1]
```diff
@@ -26,6 +26,10 @@
 ; the locale to fallback when the asked string doesn't exist in the current locale
 fallbackLocale=en_US
 
+[minimumRequiredVersion]
```
Comment (mdouchin) [inline lizmap/app/system/mainconfig.ini.php:29]: Please add a concise comment above with a `;` to explain the purpose of these new variables

**Review by mdouchin — CHANGES_REQUESTED** [review]
Thanks. Just a small change requested

**Comment by Gustry** [comment]
> The backport to `release_3_5` failed:

Manually backported

**Pull request merged**